src/theory/quantifiers/theory_quantifiers.h

```diff
@@ -19,14 +19,11 @@
 #ifndef CVC4__THEORY__QUANTIFIERS__THEORY_QUANTIFIERS_H
 #define CVC4__THEORY__QUANTIFIERS__THEORY_QUANTIFIERS_H
 
-#include "context/context.h"
 #include "expr/node.h"
-#include "theory/output_channel.h"
 #include "theory/quantifiers/proof_checker.h"
 #include "theory/quantifiers/quantifiers_rewriter.h"
+#include "theory/quantifiers/quantifiers_state.h"
 #include "theory/theory.h"
-#include "theory/valuation.h"
```

Comment:
It would be better to keep the includes for `context.h`, `output_channel.h`, and `valuation.h` because those are arguments to the constructor of `TheoryQuantifiers` for example.